Chinese/迪力.txt

```diff
@@ -0,0 +1 @@
+迪力会成为一名伟大的程序员，他正在为这个梦想不断奋斗着。继续勇往直前吧，Dilyar!
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.